doc/GettingStarted.tex

```diff
@@ -695,53 +643,48 @@ \subsubsection{Creating a bank}
 
 \subsubsection{Entering accounts}
 
-Double click or right click on the \smhtmladdimg{NewItem133.png} block
+Double click or right click on the \htmladdimg{NewItem133.png} block
 to bring up the Godley Table:
-\fwhtmladdimg{NewItem135.png} 
 
+\htmladdimg{NewItem135.png} 
 
-When a Godley Table is first loaded, it has room for one account to be
-defined. To create an additional account, click on the
-\smhtmladdimg{NewItem163.png} button above the first account. One
-click then adds another column in which an additional account can be
-defined.
 
-\fwhtmladdimg{NewItem164.png}
+When a Godley Table is first loaded, it has room for one account to be defined. To create an additional account, click on the \htmladdimg{NewItem163.png} button above the first account. One click then adds another column in which an additional account can be defined.
 
+\htmladdimg{NewItem164.png} 
 
-A column can be deleted by clicking on the \smhtmladdimg{NewItem165.png} button above the column.
+
+A column can be deleted by clicking on the \htmladdimg{NewItem165.png} button above the column.
 
 To define bank accounts in the system you enter a name into the row
-labeled ``\verb+Flows V / Stock Variables ->+''. For example, if you were
```

Comment:
stet. The character '>' typesets as an upside down ? in LaTeX. It's best to leave this as a verbatim text.